--with-docbook-dir= does not seem to have an effect

I tried to execute "make dist" on a freshly checked out git HEAD, but it failed with:
```
  GEN      doc/man/syslog-ng.8
warning: failed to load external entity "http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl"
cannot parse http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl
make[1]: *** [doc/man/syslog-ng.8] Error 4
```
Even if I have docbook.xsl at the usual location, it tires to download it from the Internet (OK, just found, it's the new default). As it turned out, SF is down:

The sourceforge.net website is temporarily in static offline mode.
Only a very limited set of project pages are available until the main website returns to service.

Well, it's off-line since last Thursday...

Back to the problem: After using the configure flag: --with-docbook-dir=/usr/share/xml/docbook/stylesheet/nwalsh5/1.78.1/ or
--with-docbook-dir=/usr/share/xml/docbook/stylesheet/nwalsh5/1.78.1/manpages/ or --with-docbook-dir=/usr/share/xml/docbook/stylesheet/docbook-xsl/ it still tries to download the file from the Internet.

--- a/configure.ac
+++ b/configure.ac
@@ -229,10 +229,10 @@ AC_ARG_WITH(systemd-journal,
 AC_ARG_WITH(python,
               [  --with-python=VERSION    Build with a specific version of python])
 
-AC_ARG_WITH(docbook,
+AC_ARG_WITH(docbook-dir,
             AC_HELP_STRING([--with-docbook-dir=DIR],
                            [compiling manpages using docbook in the specified path, if not set online version will be used from http://docbook.sourceforge.net]),
-            [ XSL_STYLESHEET=$with_docbook ],
+            [ XSL_STYLESHEET=$with_docbook_dir ],
             [ XSL_STYLESHEET=http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl ]
             )
 

